CMakeLists.txt

```diff
@@ -2,6 +2,12 @@ cmake_minimum_required(VERSION 3.12)
 
 project(RocketCode2020 C CXX)
 
+cmake_host_system_information(RESULT _memfree QUERY AVAILABLE_PHYSICAL_MEMORY)
+set_property(GLOBAL PROPERTY JOB_POOLS four_jobs=4)
+if (_memfree LESS 1000)
```

Comment:
Should this be raised to 2gb?

Reply:
The RPi we are currently using has a limit of 1GB. Did we have trouble with the other RPi?